examples/assets/Molecules_Dataset_Collection/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Not sure this file is needed - there's a repo-wide license already

Reply:
Adjusting the approach to fork the repo and git clone it